cbmex/cbmex.cpp

```diff
@@ -22,6 +22,7 @@
 
 */
 
+#include <algorithm>  // Use C++ default min and max implementation.
 #include "StdAfx.h"

```

Comment:
This (StdAfx) should be the first include (also in other places), otherwise Windows builds with MFC (like Blackrock's) will fail.